static/bidder-info/smaato.yaml

```diff
@@ -20,4 +20,7 @@ userSync:
   redirect:
     url: "https://s.ad.smaato.net/c/?adExInit=p&redir={{.RedirectURL}}&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}"
     userMacro: "$UID"
+  iframe:
+    url: "https://s.ad.smaato.net/i/?adExInit=p&redir={{.RedirectURL}}&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}"
```

Comment:
Verified the URL, `200 Ok` is returned. Also, /`setuid` is called with `uids` header set